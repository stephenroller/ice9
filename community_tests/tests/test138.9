# var a,b : int[1], c,d : bool[0]							fail

var a,b : int[1], c,d : 
						bool
						[
						0
						]		

;