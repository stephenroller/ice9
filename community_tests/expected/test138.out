line 6: array dimensions must be > 0